docs/deprecated.md

```diff
@@ -0,0 +1,41 @@
+# Deprecated functionality
+
+Deprecated features are features that sigfox-iotagent stills support but that are not maintained or evolved any longer. In
+particular:
+
+-   Bugs or issues related with deprecated features and not affecting any other feature are not addressed (they are
+    closed in github.com as soon as they are spotted).
+-   Documentation on deprecated features is removed from the repository documentation. Documentation is still available
+    in the documentation set associated to older versions (in the repository release branches).
+-   Deprecated functionality is eventually removed from sigfox-iotagent. Thus you are strongly encouraged to change your
+    implementations using sigfox-iotagent in order not rely on deprecated functionality.
+
+A list of deprecated features and the version in which they were deprecated follows:
+
+-   Support to NGSI v1.
+-   Support to Node.js v10 in sigfox-iotagent 1.5.0. The use of Node.js v12 is highly recommended.
```

Comment:
Looking to https://github.com/telefonicaid/sigfox-iotagent/releases I think the following is more accurate:

```suggestion
-   Support to Node.js v4 and v6 in sigfox-iotagent 1.0.0 (finally removed in 1.1.0)
-   Support to Node.js v8 in sigfox-iotagent 1.4.0 (finally removed in 1.5.0)
-   Support to Node.js v10 in sigfox-iotagent 1.5.0. The use of Node.js v12 is highly recommended.
```